src/main.js

```diff
@@ -20,8 +20,12 @@ pkg.require({
     HackToolbox: '0',
 });
 
+const GLib = imports.gi.GLib;
 const {HackToolboxApplication} = imports.app;
 
 function main(argv) {
-    return (new HackToolboxApplication()).run(argv);
+    let app = new HackToolboxApplication();
+    if (GLib.getenv('HACK_TOOLBOX_PERSIST'))
+        app.hold();
```

Comment:
Seems fine as a quick hack even though we never release it. It's unfortunate the app class is hardcoded to have the IS_SERVICE flag set, but fixing that would be more trouble than this small PR warrants.

Reply:
Yeah, that was too much yak shaving for something this simple. We use this pattern in other similar apps like the discovery feed.